### PR TITLE
FOGL-7084 CentOS Stream 9 support

### PR DESCRIPTION
**Pull request by Mohit04tomar**
Changes in this PR is done so that installation of fledge through RPM package can be done properly on CentOS Stream 9.

### CODE REVIEW

**Review by MarkRiddoch — DISMISSED** [review]

**Review by YashTatkondawar — APPROVED** [review]

File: packages/RPM/SPECS/fledge.spec [hunk 1]
```diff
@@ -502,6 +523,7 @@ fi
 
 set -e
 PKG_NAME="fledge"
+OS_VERSION=$(cat /etc/os-release | grep 'VERSION_ID=' | cut -f2 -d= | sed 's/"//g')
```
Comment (ashish-jabble) [inline packages/RPM/SPECS/fledge.spec:526]: unused?

File: packages/RPM/SPECS/fledge.spec [hunk 2]
```diff
@@ -71,6 +75,7 @@ Fledge, the open source platform for the Internet of Things
 set -e
 
 PKG_NAME="fledge"
+OS_VERSION=$(cat /etc/os-release | grep 'VERSION_ID=' | cut -f2 -d= | sed 's/"//g')
```
Comment (ashish-jabble) [inline packages/RPM/SPECS/fledge.spec:78]: unused?

File: packages/RPM/SPECS/fledge.spec [hunk 3]
```diff
@@ -15,7 +15,11 @@ VCS:           __VCS__
 %define install_path	/usr/local
 
 Prefix:        /usr/local
+%if ( 0%{?centos} >= 7 && 0%{?centos} < 8 ) || ( 0%{?rhel} >= 7 && 0%{?rhel} < 8 )
```
Comment (ashish-jabble) [inline packages/RPM/SPECS/fledge.spec:18]: Can't we simplify with <9 check?

File: packages/RPM/SPECS/fledge.spec [hunk 4]
```diff
@@ -324,7 +330,12 @@ copy_fledge_sudoer_file() {
 }
 
 copy_service_file() {
-    cp /usr/local/fledge/extras/scripts/fledge.service /etc/init.d/fledge
+    if [[ ${OS_VERSION} == *"7"* ]]
+    then
+        cp /usr/local/fledge/extras/scripts/fledge.service /etc/init.d/fledge
+    else
+        cp /usr/local/fledge/extras/scripts/fledge.service /etc/rc.d/init.d//fledge
```
Comment (ashish-jabble) [inline packages/RPM/SPECS/fledge.spec:337]: Double slashes at the end

**Review by ashish-jabble — APPROVED** [review]

**Pull request merged**